docs/attrEnd.md

```diff
@@ -1,7 +1,8 @@
 @function can-view-parser.ParseHandler.attrEnd attrEnd
 @parent can-view-parser.ParseHandler
-@signature `attrEnd(attrName)`
+@signature `attrEnd(attrName, lineNo)`
 
 Called at the end of parsing an attribute; after the [can-view-parser.ParserHandler.attrStart] and [can-view-parser.ParserHandler.attrValue] functions have been called.
 
 @param {String} attrName The name of the attribute.
+@param {Number} lineNo The line number of the error (undefined in production).
```

Comment:
Should these be something like?

> The line number of the attribute to be used in error messages (undefined in production).

The `lineNo` isn't specific to errors per se.

Reply:
yes, yes they should